packages/terra-site/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Fixed stylelint issues
 
 2.6.0 - (April 15, 2018)
```

Comment:
I thought we weren't releasing site anymore?

Reply:
Haven't added it to the ignore config in lerna.json yet so it is still being published. I can update that in a separate PR.